org.eclipse.xtext.ide.tests/src/org/eclipse/xtext/ide/tests/server/DocumentHighlightTest2.xtend

```diff
@@ -0,0 +1,47 @@
+/*******************************************************************************
+ * Copyright (c) 2016 TypeFox GmbH (http://www.typefox.io) and others.
```

Comment:
wrong date

Reply:
Fixed date & updated copyright to Sigasi.